### DISCUSSION TITLE
{{ Current User.email}} Binding is not updating

### DISCUSSION THREAD

**Discussion by LifeofGlen** [Budibase/budibase#14424] [Help] [answered]
I have a self hosted version of BB and for some reason, the current user binding always stays as the person who created/published app and doesn't show the current logged in user.

I've tried a few things, but can't seem to get it working.

I'm happy to do a video/export or provide whatever is needed to further debug.

Any suggestions why this might not be working would also be appreciated.

![image](https://github.com/user-attachments/assets/2bdbaaf5-923c-4f72-9af9-f85ec165600d)

Ive added just a text headline using this binding to test/debug, but as mentioned, it always stays the same.

**Comment by ConorWebb96** [comment]
Hey @LifeofGlen,

Have you logged out and in as a different user?

**Reply by LifeofGlen** [reply, marked as answer]
I did a few times actually. I eventually figured out that i cant use this binding as a default value when updating a row if there is already a value in the row. I had to add that column as a seperate field to update on the save row action. so this was sorted, thanks.
